packages/story-editor/src/app/media/local/useContextValueProvider.js

```diff
@@ -263,6 +308,15 @@ export default function useContextValueProvider(reducerState, reducerActions) {
     media?.forEach((mediaElement) => postProcessingResource(mediaElement));
   }, [media, mediaType, searchTerm, postProcessingResource]);
 
+  useEffect(() => {
+    (async () => {
+      for (const mediaElement of media) {
+        // eslint-disable-next-line no-await-in-loop
+        await postProcessingBlurHashResource(mediaElement);
+      }
+    })();
```

Comment:
awaits in loop is really bad for performance.

Hence my other comment above for not making `processMediaBlurhash` async.

Then this can be changed to a simple `media?.forEach((mediaElement) => postProcessingBlurHashResource(mediaElement));`

Reply:
If we don't wait for each one to complete, then they all fire at once.

Reply:
`processMediaBlurhash` makes sure that only one is being processed at a time.